### ISSUE TITLE
Tests are taking relatively long

### ISSUE THREAD

**Issue by lubianat** [lubianat/wdcuration#14]
Current tests are taking relatively long (when running base "pytest"); can I run it only for a subset of the tests ? (I guess so @jvfe) 

Maybe there are other ways of shortening the run

**Comment by jvfe** [comment]
There will be some tests which will inevitably take longer because the underlying function uses network requests. 

In the functions where network requests are not a big part of the logic we can mock the network requests to test the actual revelant part of the function - for example, we could mock the return value for search_wikidata inside of add_key, since the revelant part of add_key (I think) is being able to handle the user input (y/n/skip/etc).

All that being said, I don't actually think the tests are taking that long - they're running in 15s or under here.

**Comment by lubianat** [comment]
Yep, I understand, they are naturally long. 15s though can be a bit long for TDD, specially when refactoring stuff

**Comment by jvfe** [comment]
Oh and sorry, answering your question: You can indeed run only a subset of the tests, the way I usually do it with pytest is by keyword search (-k), e.g.:

```
pytest -k TestWdcurationSPARQL -v
```
